phs.ttl

```diff
@@ -252,17 +298,16 @@
        rdfs:subClassOf dwc:Taxon .
 
 
-###  http://linked.data.gov.au/def/phs#SampleAnimal
-:SampleAnimal rdf:type owl:Class ;
+###  http://linked.data.gov.au/def/phs#Pest
+:Pest rdf:type owl:Class ;
               rdfs:subClassOf dwc:Taxon ,
                               sosa:Sample ,
```

Comment:
@F3RR3T and Simon Opper observed that the sosa:Sample should be removed from this subclass of taxon. We need to address this in a later refactor